test/unit/common/test_pipeline.cpp

```diff
@@ -65,17 +36,18 @@ TEST(Pipeline, BasicWithSenderAdaptors) {
   auto guard0 = serial();
   auto guard1 = serial();
 
-  EXPECT_FALSE(guard0.is_ready());
-  EXPECT_FALSE(guard1.is_ready());
+  // TODO: What would be a reasonable test here now that guard0 is not a future?
```

Comment:
This has to be updated with a reasonable new test. Senders don't have `is_ready` in general.